tilelang/utils/target.py

```
@@ -238,29 +238,20 @@ def determine_target(target: TargetLike | Literal["auto"] = "auto", return_objec
 
     return_var: str | TargetConfig | Target = target
 
+    if getattr(target, "_tilelang_target_marker", False):
+        resolved_target = target.to_tvm_target()
+        return resolved_target if return_object else resolved_target
```

Ruff (0.15.15)

[warning] Useless `if`-`else` condition

(RUF034)